doc/contributing/api-documentation.md

```diff
@@ -4,9 +4,8 @@ The Node.js API documentation is generated by an in-house tooling that resides
 within the [tools/doc](https://github.com/nodejs/node/tree/main/tools/doc)
 directory.
 
-The build process (using `make doc`) uses this tooling to parse the markdown
-files in [doc/api](https://github.com/nodejs/node/tree/main/doc/api) and
-generate the following:
+The build process (using `make doc` or `make doc-only`) uses this tooling to
+parse the markdown files in [doc/api][] and generate the following:
```

Comment:
While we're changing it, I suggest making it easier to understand this is a path
```suggestion
parse the markdown files in [`doc/api/`][] and generate the following:
```

Reply:
You didn't want to add the trailing slash?

Reply:
Gah! Sorry, my eyes skipped right over that. Added.